### PR TITLE
LIWO-529: mouse-over info

### PR DESCRIPTION
**Pull request by sjoerdbeentjes**

### CODE REVIEW

File: src/lib/leaflet-utils/layer-factory.js [hunk 1]
```diff
@@ -11,7 +11,19 @@ import { greyIcon, redIcon, defaultIcon, iconsByLayerType } from '@/lib/leaflet-
...
+  var handlers = (vnode.data && vnode.data.on) ||
+    (vnode.componentOptions && vnode.componentOptions.listeners)
+
+  if (handlers && handlers[name]) {
+    handlers[name].fns(data)
+  }
+}
+
+export default async function createLayer (layer, { onClick }, _, vnodeRef) {
+  vnode = vnodeRef
```
Comment (SiggyF) [inline src/lib/leaflet-utils/layer-factory.js:26]: can you explain what's going on with this vnode/vnodeRef?

File: src/lib/leaflet-utils/layer-factory.js [hunk 2]
```diff
@@ -72,19 +84,23 @@ function createCluster (layer, onClick) {
...
   // TODO: implement is  controllable
   marker.on('click', (evt) => {
     evt.layer = layer
     onClick(evt)
   })
   marker.on('mouseover', (event) => {
+    marker.bindTooltip(name)
+
+    // emit properties so e.g. state can be used to set tooltip text
+    emit(vnode, 'marker:mouseover', { feature, marker })
```
Comment (SiggyF) [inline src/lib/leaflet-utils/layer-factory.js:97]: Do I understand it correctly that a regular `div` now becomes a vue component?

**Review by SiggyF — APPROVED** [review]
Should we transform  v-leaflet to a component?

File: src/lib/leaflet-utils/layer-factory.js (hunk 1 above)
Comment (sjoerdbeentjes) [reply src/lib/leaflet-utils/layer-factory.js:30]: I added comments in the code to clarify on what's going on in my commit after this comment.

**Pull request merged**